docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -777,3 +790,36 @@ Keen Eye:
     Mechanic: When you make a Detection check to observe the world around you (visual, auditory or olfactory), draw with the Upper Hand.
     Type: Passive
     XP: 1
+
+_Discard Note: # Not sure how this will render
+    Category: Discard
+    Mechanic: A character is limited to one Discard Power
+    Type: Passive
+
+Aggro:
+    Category: Discard
+    Description: Protecting your friends means taking the big hits.
+    Mechanic: Discard N cards to force all creatures within 2 spaces to make their next Check to attack targets besides the triggering Dealer at Lower Hand (N/2)
+    Type: Minor
+    XP: 0 
+
+Help:
+    Category: Discard
+    Description: You're there just when your friends need you.
+    Mechanic: Discard N cards to grant an ally within 6 spaces Upper Hand (N/2) on all Checks for their next turn
+    Type: Minor
+    XP: 0 
+
+Thwart:
+    Category: Discard
+    Description: You have a few tricks up your sleeve for drawing the enemy's attention.
+    Mechanic: Discard N cards and cause an enemy within 6 spaces to make all Checks at Lower Hand (N/2) until the start of the triggering Dealer's next turn
+    Type: Minor
+    XP: 0 
+
+Intensify:
+    Category: Discard
+    Description: An explosive attack might mean running out of gas.
+    Mechanic: Discard no cards. Your next attack is treated as an Epic Event. Discard as many cards as you choose until a Major Success. Standard Successes are treated as Misses
+    Type: Minor
+    XP: 0 
```

Comment:
Should we add `Prereq: Level: 2` to these? Anything else we'd need to add as well?

Reply:
Sure, I'll add prereq level. Not sure how to best add a 'custom limitation' besides the note above.